packages/frontend/src/providers/DashboardProvider.tsx

```diff
@@ -244,7 +244,7 @@ export const DashboardProvider: React.FC = ({ children }) => {
     });
 
     useEffect(() => {
-        const newParams = new URLSearchParams();
+        const newParams = new URLSearchParams(search);
```

Comment:
Had to fix this so this hook updates the URL params instead of completely replacing them.